app/src/main/java/org/sil/gatherwords/DatabaseAccess.java

```diff
@@ -0,0 +1,49 @@
+package org.sil.gatherwords;
+
+import android.os.AsyncTask;
+
+import org.sil.gatherwords.room.AppDatabase;
+import org.sil.gatherwords.room.Session;
+import org.sil.gatherwords.room.SessionDao;
+import org.sil.gatherwords.room.Word;
+import org.sil.gatherwords.room.WordDao;
```

Comment:
It does not appear that you have committed any of these classes.

Reply:
At the time, the schema branch was still pending. I tried to branch  from master and merge with the schema branch, but it seems like most things got lost. What's strange is that those files are marked as unchanged compared to the branch on my end.